MediaBrowser.Controller/MediaEncoding/IMediaEncoder.cs

```diff
@@ -122,10 +151,24 @@ public interface IMediaEncoder : ITranscoderSupport
         /// <returns>System.String.</returns>
         string EscapeSubtitleFilterPath(string path);
 
+        /// <summary>
+        /// Sets the path to find FFmpeg.
+        /// </summary>
         void SetFFmpegPath();
 
+        /// <summary>
+        /// Updated the encoder path.
```

Comment:
```suggestion
        /// Updates the encoder path.
```

Reply:
Many thanks for making this a "suggestion".